.github/workflows/go-ci/action.yml

```diff
@@ -14,6 +14,11 @@ runs:
         working-directory: ${{ github.workspace }}/_build/install/default/share/go/src
         args: --config=${{ github.workspace }}/.golangci.yml
 
+    - name: Run Go Tests
```

Comment:
I think the test already run on https://github.com/xapi-project/xen-api/blob/c1594b99e119a0cbfae261e5fc4cedafc90d9742/ocaml/sdk-gen/component-test/run-tests.sh#L20 ? Is this cover your new test case?

Reply:
as I understand it, `go test-v &` as shown above will only run the tests for `jsonrpc-client`

I have purposedly decided not to add the tests in this PR there, since they are not integration tests with live hosts, but rather pure unit tests

Reply:
OK, make sense.